ATM_project/MainForm.cs

```diff
@@ -54,33 +59,84 @@ public MainForm()
             text_on_ATM.Parent = ATM;
             text_on_ATM.BackColor = Color.Transparent;
             this.BackColor = Color.FromArgb(200, 200, 200);
+                    
+            conn.Open();
+            
+            string query = "Select COUNT(Code) FROM cardtable";
+            SqlCommand cmd = new SqlCommand(query, conn);            
+            SqlDataReader reader1;
+            reader1 = cmd.ExecuteReader();
+
+            if (!reader1.Read())
+            {                               
+                query = "INSERT INTO cardtable(Code,PIN,Amount) values('Cod1','PIN1','" + (decimal)1600.00 + "') " +
+                "INSERT INTO cardtable(Code,PIN,Amount) values('Cod2','PIN2','" + (decimal)1750.00 + "') ";
+                cmd = new SqlCommand(query, conn);
+                cmd.ExecuteNonQuery();
+            }
+            conn.Close();
+            conn.Open();
+            query = "Select * FROM bank WHERE Id=1";
+            SqlCommand cmd2 = new SqlCommand(query, conn);
+            SqlDataReader reader2;
+            reader2 = cmd2.ExecuteReader();
+
+            if (!reader2.Read())
+            {
+                conn.Close();
+                conn.Open();
+                query = "INSERT INTO bank(Id,BankMoney) values(1,'"+ (decimal)1000000.00 +"')";
+                //query = "UPDATE bank SET BankMoney = " + (decimal)1000000.00 + " WHERE Id=1;";
+                cmd = new SqlCommand(query, conn);
+                cmd.ExecuteReader();
+                bankMoney = (decimal)1000000.00;
+            }
+            else
+            {
+                conn.Close();
+                conn.Open();
+                string query3 = "Select BankMoney From bank Where Id=1";
```

Comment:
Create a new method for this check.
Validate it for the case in which you don't have any data.